app/models/setting/provisioning.rb

```diff
@@ -63,10 +63,10 @@ def self.default_settings
       self.set('default_pxe_item_global', N_("Default PXE menu item in global template - 'local', 'discovery' or custom, use blank for template default"), nil, N_("Default PXE global template entry")),
       self.set('default_pxe_item_local', N_("Default PXE menu item in local template - 'local', 'local_chain_hd0' or custom, use blank for template default"), nil, N_("Default PXE local template entry")),
       self.set(
-        'disassociate_vms_before_host_deletion',
-        N_("Disassociate VMs from Hosts prior deletion. When enabled, VMs linked to Hosts will not be deleted on Compute Resource, meaning they can be re-associated or imported back to Foreman again. This does not automatically power off the VM"),
-        false,
-        N_("Disassociate VMs before Host deletion")
+        'destroy_vm_on_host_delete',
+        N_("Destroy associated VM on host delete. When enabled, VMs linked to Hosts will not be deleted on Compute Resource, meaning they can be re-associated or imported back to Foreman again. This does not automatically power off the VM"),
```

Comment:
"When enabled, VMs linked to Hosts will not be deleted on Compute Resource" - I'd expect to have "**will be deleted**" instead.

Reply:
thanks, good catch :-)